warp10/src/main/java/io/warp10/standalone/StandaloneIngressHandler.java

```diff
@@ -357,20 +361,30 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
         
         if (dr.getAttributesSize() > 0) {
           if (null != dr.getAttributes().get(StandaloneAcceleratedStoreClient.ATTR_NOCACHE)) {
-            nocache = true;
-          } else {
-            nocache = false;
+            forcedNocache = true;
+            nocache = !"false".equals(dr.getAttributes().get(StandaloneAcceleratedStoreClient.ATTR_NOCACHE));
           }
           if (null != dr.getAttributes().get(StandaloneAcceleratedStoreClient.ATTR_NOPERSIST)) {
-            nopersist = true;
-          } else {
-            nopersist = false;
+            forcedNopersist = true;
+            nopersist = !"false".equals(dr.getAttributes().get(StandaloneAcceleratedStoreClient.ATTR_NOPERSIST));
```

Comment:
In DELETE, FETCH and UPDATE, the test is `Boolean.TRUE.equals` so for coherency it should be `"true".equals`.